src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/roboticssupport/RoboticsNotificationServiceImpl.java

```diff
@@ -0,0 +1,169 @@
+import uk.gov.hmcts.cmc.claimstore.events.claim.DocumentGenerator;
+import uk.gov.hmcts.cmc.claimstore.events.paidinfull.PaidInFullEvent;
+import uk.gov.hmcts.cmc.claimstore.events.response.DefendantResponseEvent;
+import uk.gov.hmcts.cmc.claimstore.events.response.MoreTimeRequestedEvent;
+import uk.gov.hmcts.cmc.claimstore.idam.models.GeneratePinResponse;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.claimstore.rpa.DefenceResponseNotificationService;
+import uk.gov.hmcts.cmc.claimstore.rpa.MoreTimeRequestedNotificationService;
+import uk.gov.hmcts.cmc.claimstore.rpa.PaidInFullNotificationService;
+import uk.gov.hmcts.cmc.claimstore.rpa.RequestForJudgementNotificationService;
+import uk.gov.hmcts.cmc.claimstore.services.ClaimService;
+import uk.gov.hmcts.cmc.claimstore.services.ResponseDeadlineCalculator;
+import uk.gov.hmcts.cmc.claimstore.services.UserService;
+import uk.gov.hmcts.cmc.domain.exceptions.BadRequestException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+@Service
+public class RoboticsNotificationServiceImpl implements RoboticsNotificationService {
+
+    private static final String RPA_STATE_MISSING = "missing";
+    private static final String RPA_STATE_INVALID = "invalid";
+    private static final String RPA_STATE_FAILED = "failed";
+    private static final String RPA_STATE_SUCCEEDED = "succeeded";
+
+    private final UserService userService;
+
+    private final ClaimService claimService;
+    private final MoreTimeRequestedNotificationService moreTimeRequestedNotificationService;
+    private final DefenceResponseNotificationService defenceResponseNotificationService;
+    private final RequestForJudgementNotificationService ccjNotificationService;
+    private final PaidInFullNotificationService paidInFullNotificationService;
+    private final ResponseDeadlineCalculator responseDeadlineCalculator;
+
+    private final AppInsightsExceptionLogger appInsightsExceptionLogger;
+    private final DocumentGenerator documentGenerator;
+
+    @Autowired
+    public RoboticsNotificationServiceImpl(ClaimService claimService, UserService userService,
+                                           MoreTimeRequestedNotificationService moreTimeRequestedNotificationService,
+                                           DefenceResponseNotificationService defenceResponseNotificationService,
+                                           RequestForJudgementNotificationService ccjNotificationService,
+                                           PaidInFullNotificationService paidInFullNotificationService,
+                                           ResponseDeadlineCalculator responseDeadlineCalculator,
+                                           AppInsightsExceptionLogger appInsightsExceptionLogger,
+                                           DocumentGenerator documentGenerator) {
```

Comment:
_De facto_ standard for method signatures that can't fit on one line is:
* new line after `(`
* each param on its own line
* each param indented a further 4 spaces
* `) {` on its own line aligned with the start of the method

Reply:
done